src/endpoint/Token.Exchange/Contract/TokenExchangeOut.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using GarageGroup.Infra;
+
+namespace GGroupp.Yandex.Migration;
+
+public sealed record class TokenExchangeOut
+{
+    [JsonBodyOut]
+    public string IamToken { get; init; } = string.Empty;
```

Comment:
Use required and you will not need "= string.Empty;":
public required string IamToken { get; init; }